torch/_dynamo/variables/misc.py

```diff
@@ -790,6 +790,16 @@ def wraps(fn):
             return variables.lists.DequeVariable(
                 items, mutable_local=MutableLocal(), **options
             )
+        elif self.value is functools.partial:
+            assert len(args) > 0
```

Comment:
A user can trigger this with a malformed functools.partial call, therefore this should be unimplemented not assert

Reply:
Okay